code/controlconfig/controlsconfigcommon.cpp

```diff
@@ -612,6 +613,9 @@ void control_config_common_init()
 		Scan_code_text = Scan_code_text_english;
 		Joy_button_text = Joy_button_text_english;
 	}
+
+	auto current = io::joystick::getCurrentJoystick();
```

Comment:
`getCurrentJoystick` can return `nullptr` if there is no joystick. #773 fixed a similar issue so you should make sure that `current` is not `nullptr` when you use it.